src/Symfony/Bundle/FrameworkBundle/CacheWarmer/ConfigBuilderCacheWarmer.php

```diff
@@ -68,19 +69,31 @@ public function warmUp(string $cacheDir, ?string $buildDir = null): array
             }
         }
 
+        $configurations = [];
         foreach ($extensions as $extension) {
+            if (null === $configuration = $this->getConfigurationFromExtension($extension)) {
+                continue;
+            }
+
+            $alias = lcfirst(str_replace('_', '', ucwords($extension->getAlias(), '_')));
```

Comment:
I think functions should use underscore names like in PHP, so `web_profiler()` and not `webProfiler()`.

Reply:
Here you go, this is how `WebProfiler` configuration function looks like:

```php
function web_profiler(
    /** @var array{
     *     toolbar?: array{ // Profiler toolbar configuration
     *         enabled?: bool, // Default: false
     *         ajax_replace?: bool, // Replace toolbar on AJAX requests Default: false
     *     },
     *     intercept_redirects?: bool, // Default: false
     *     excluded_ajax_paths?: string|int|float|bool, // Default: "^/((index|app(_[\\w]+)?)\\.php/)?_wdt"
     * } $config */
    array $config = []): \Closure {
    return static function (\Symfony\Config\WebProfilerConfig $webProfilerConfig) use ($config) {
        $webProfilerConfig->configure($config);
    };
}
```